### ISSUE TITLE
See what your friends are watching

### ISSUE THREAD

**Issue by itsMatoosh** [Stremio/stremio-features#63]
Hi, I had an idea, that it would be really cool to see what movies/series the user's facebook friends are currently watching. This feauture is really well implemented on Spotify and I think it would work great in Stremio as well.

**Comment by Ivshti** [comment]
Indeed, it could be an engaging feature. We are considering social features anyway, so something like that would certainly come at some point

thanks for suggesting!

**Comment by victorgveloso** [comment]
There is a stremio addon that add support for [discord rich present](https://www.reddit.com/r/StremioAddons/comments/dgcezb/local_addon_discord_rich_presence_for_stremio/). It might be enough for most users.

**Comment by TRtomasz** [comment]
Social features will not be added at this point
